sentences/de/_common.yaml

```
@@ -69,9 +69,9 @@ lists:
         out: "lock"
   lock_states:
     values:
-      - in: "(([ab|zu]ge|ver)schlossen|verriegelt)"
+      - in: "(([ab|zu]ge|ver)schlossen|verriegelt|((ab|zu)ge|ver)sperrt)"
         out: "locked"
-      - in: "(nicht ([ab|zu]ge|ver)schlossen|entriegelt)"
+      - in: "(nicht (([ab|zu]ge|ver)schlossen|verriegelt|((ab|zu)ge|ver)sperrt)|entriegelt|entsperrt)"
```

```suggestion
      - in: "(nicht (([ab|zu]ge|ver)(schlossen|sperrt)|verriegelt)|ent(riegelt|sperrt))"
```
this allows:

- nicht abgeschlossen
- nicht zugeschlossen
- nicht geschlossen
- nicht verschlossen
- nicht abgesperrt
- nicht zugesperrt
- nicht gesperrt
- nicht versperrt
- nicht verriegelt
- entriegelt
- entsperrt